### PR TITLE
fix: unicode escaping by replacing fastjson version

### PR DESCRIPTION
**Pull request by devsergiy**
## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

**Comment by jensneuse** [comment]
@devsergiy can you add a test that validates that the behavior is "locked in*?

**Review by StarpTech — CHANGES_REQUESTED** [review]
Please add a test case.

**Comment by StarpTech** [comment]
You can take inspiration from 

- https://github.com/golang/go/blob/master/src/encoding/json/encode_test.go#L784
- https://github.com/goccy/go-json/blob/master/encode_test.go#L928

File: router-tests/testdata/queries/products.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         "clientLanguages": [
           "RUST",
           "TYPESCRIPT"
-        ]
+        ],
+        "unicode": "😀♥!\u0015\u0010🝥"
```
Comment (StarpTech) [inline router-tests/testdata/queries/products.json:26]: This won't reproduce the error. You can use `"\u0015"`

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: router-tests/testdata/queries/products.json (hunk 1 above)
Comment (devsergiy) [reply router-tests/testdata/queries/products.json:26]: you should have tried
```
Unicode: "\U0001F600\u2665\u0021\u0015\u0010\U0001F765"
```

Comment (StarpTech) [reply router-tests/testdata/queries/products.json:26]: 👍

Comment (StarpTech) [reply router-tests/testdata/queries/products.json:26]: I didn't understand the test in the first place. You extended the test query and updated the fixture. Now, I understand. Thanks.

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**